test/functional/columns/value-transformer/value-transformer.ts

```diff
@@ -54,5 +55,17 @@ describe("columns > value-transformer functionality", () => {
 
     })));
 
+    it("should apply three transformers", () => Promise.all(connections.map(async connection => {
+        const userRepository = await connection.getRepository(User);
+        const email = `${connection.name}@JOHN.doe`;
+        const user = new User();
+        user.email = email;
+
+        await userRepository.save(user);
+
+        const dbUser = await userRepository.findOne();
+        dbUser && dbUser.email.should.be.eql(email.toLocaleLowerCase());
```

Comment:
What if there is a bug and only first value transformer (lowercase in this case) is applied ? :) Also maybe add test with empty array - for sure it will not throw errors or something unexpected in future.

Reply:
Good thinking, I'll improve the tests